CMSIS/NN/Source/ConvolutionFunctions/arm_convolve_1x1_HWC_q7_fast_nonsquare.c

```diff
@@ -159,9 +159,9 @@ arm_status arm_convolve_1x1_HWC_q7_fast_nonsquare(const q7_t * Im_in,
 
                 pA = read_and_pad_reordered(pA, &inA1, &inA2);
 
-                inB1 = *__SIMD32(pB)++;
+                inB1 = arm_nn_read_q15x2_ia(pB);
```

Comment:
arm_nn_read_q15x2_ia expects a double pointer and single pointer is passed for more of this PR, Please recheck and re-compile.

Reply:
Good finding. It is done